draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -178,7 +177,7 @@ If a provider has an application-specific need to have multiple verifications fo
 
 This again allows the provider to query only for application-specific records it needs, while giving flexibility to the user adding the DNS verification record (i.e. they can be given permission to only add records under a specific prefix by the DNS administrator). Whether or not multiple verifying records can exist for the same domain is up to the implementation.
 
-Providers MUST provide clear instructions on when a verifying record can be removed. The user SHOULD de-provision the resource record(s) provisioned for a DNS-based domain verification challenge once the challenge is complete.
+Providers MUST provide clear instructions on when a verifying record can be removed. As a corollary, if the provider periodically re-validates the domain, then the provider MUST clarify this and the time after which it does so.  The user SHOULD de-provision the resource record(s) provisioned for a DNS-based domain verification challenge once the challenge is complete in order to prevent bloat.
```

Comment:
I don't think this text is clear.

It is also very different to use normative language to convey the time period for the record (which will be lost between various parties conveying this) and using clear language _within_ the rdata so it becomes obvious when it expires or whether it should remain there as continuous proof of domain ownership

Reply:
I think this is now resolved in the Metadata section of the text: https://github.com/ietf-wg-dnsop/draft-ietf-dnsop-domain-verification-techniques/blob/main/draft-ietf-dnsop-domain-verification-techniques.md#metadata-for-expiry-metadata